Jenkinsfile.cloud

```diff
@@ -1,14 +1,7 @@
 // this var conveniently refers to a location on the server as well as the local dir we sync to/from
 def output = "${env.ARTIFACT_SERVER_DIR}/images"
 
-def ref
-if (env.BUILD_TYPE == 'origin') {
-    ref = "origin/3.10/x86_64/os"
-} else if (env.BUILD_TYPE == 'rhcos') {
-    ref = "openshift/3.10/x86_64/os"
-} else {
-    assert false
-}
+def ref = "openshift/3.10/x86_64/os";
```

Comment:
There's some similar logic in the other Jenkinsfiles we may want to delete as well.

Reply:
I didn't see anything in the other `Jenkinsfiles`?  But I did find other things to fix like the `README.md`, done below.

Reply:
I meant for the treecompose one, e.g. something like:

```groovy
-if (env.BUILD_TYPE != 'origin' && env.BUILD_TYPE != 'rhcos') {
-    assert false
-}
-def manifest = "host-${env.BUILD_TYPE}.json"
+def manifest = "host-rhcos.json"
```

?